src/Symfony/Component/ErrorHandler/Tests/ErrorHandlerTest.php

```diff
@@ -398,7 +398,7 @@ public function testHandleDeprecation()
             $this->assertArrayHasKey('exception', $context);
             $exception = $context['exception'];
             $this->assertInstanceOf(\ErrorException::class, $exception);
-            $this->assertSame('User Deprecated: Foo deprecation', $exception->getMessage());
+            $this->assertSame('User Deprecated: Foo deprecation at foo.php line 12', $exception->getMessage());
```

Comment:
I can understand that we might want to see that information in our logs, but encoding it into the _exception_ message does not feel right.

Reply:
I am indeed afraid adding file and line in the message could make deduplication or "grouping" harder.

Reply:
If there is a problem with exception Message then i can move the addition just before sending to the logger and let this test as it was before